bot.py

```diff
@@ -31,4 +31,41 @@ async def create_repo(ctx, repoName): #!git create_repo repo1
 	repo = user.create_repo(repoName)
 	await ctx.send("repository "+repoName+ " created!")
 
+
+# display open and closed issues
```

Comment:
Make this a [docstring](https://www.datacamp.com/community/tutorials/docstrings-python)